openff/toolkit/topology/topology.py

```diff
@@ -853,24 +838,112 @@ def from_dict(cls, d):
 # =============================================================================================
 
 
-class TopologyVirtualParticle(TopologyVirtualSite):
-    def __init__(self, virtual_site, virtual_particle, topology_molecule):
+class TopologyVirtualParticle(Serializable):
```

Comment:
`TopologyVirtualParticle` probably shouldn't inherit from `TopologyVirtualSite` otherwise it exposes properties like `particles`, so here we split the two classes